jsonHelper.js

```diff
@@ -0,0 +1,15 @@
+function stringifyJson(x) {
+  try {
+    return JSON.stringify(x);
+  } catch (error) {
+    showError(STRINGIFY_MESSAGE);
```

Comment:
You didnt understand me. This is a json helper, so show error doesnt belong here. What you need to do is import the file that has the showError function in this file.
It would be best if you choose exactly what function and variables you export from a file.
https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export

Here is an example of simple js file:
`
function1 ...
function2 ...

module.exports = {
    function1:  function1,
    function2: function2
};`

And then in the second file where you want to use this functions you will use
`const someFunctions= require('../location/of/your/file/filenameWithoutExtension');
`